daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/DevIT.scala

```diff
@@ -15,13 +15,7 @@ import org.scalatest.Inside
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
 
-// TODO(#17366): Once daml3-script diverges from script, V1DevIT and V2DevIT may not be able to
-//     share the same code anymore.
-class DevITV1 extends DevIT(LanguageMajorVersion.V1)
-
-// TODO(https://github.com/digital-asset/daml/issues/17812): re-enable this test and control its run
-//  at the bazel target level.
-//class DevITV2 extends DevIT(LanguageMajorVersion.V2)
+class DevITV2 extends DevIT(LanguageMajorVersion.V2)
```

Comment:
We ca delete this class altogether and instead define a value in the the DevIT body
```
override val majorLanguageVersion: LanguageMajorVersion = LanguageMajorVersion.V2
```
Then the same in the other classes of this kind

Reply:
Yes, these classes made sense when we had two major versions. I'll have another pass at cleaning them up. (There are others not touched by this PR.)